tests/mgbench/benchmark.py

```diff
@@ -224,6 +224,13 @@ def filter_benchmarks(generators, patterns):
                     ):
                         current[group].append((query_name, query_func))
             if len(current) > 0:
```

Comment:
This way you can reduce indentation
```suggestion
            if len(current) == 0:
              continue
```